lib/amazon_ssa_support/ssa_queue_extractor.rb

```diff
@@ -63,7 +63,10 @@ def do_extract(req)
       @ssaq.delete_request(req)
       extract_reply = @ssaq.new_reply(req)
       begin
-        ec2_vm = MiqEC2Vm.new(req[:ec2_id], @my_instance, @ec2)
+        ost = OpenStruct.new
+        ost.scanData = req[:scan_data]
```

Comment:
Do we need `ost` at all?

Reply:
Yes, we do. It contains all scan items we set in analysis profiles.